### ISSUE TITLE
Please fix podsgrant's support for xina.

### ISSUE THREAD

**Issue by Crvesoft** [LNSSPsd/PodsGrant#58]
I sincerely hope that podsgrant can support xina, and please fix it when you have time. Thank you very much.

**Comment by Gkotik** [comment]
you can, just download rootless ver (i use 0.2.3), that unpack deb in filza, open control as a text, change arm64 to arm, that make a deb and install

**Comment by Crvesoft** [comment]
> 

There is no pop-up window in this installation

**Comment by Crvesoft** [comment]
> you can, just download rootless ver (i use 0.2.3), that unpack deb in filza, open control as a text, change arm64 to arm, that make a deb and install

There is no pop-up window after installation according to your method.

**Comment by Gkotik** [comment]
> > you can, just download rootless ver (i use 0.2.3), that unpack deb in filza, open control as a text, change arm64 to arm, that make a deb and install
> 
> There is no pop-up window after installation according to your method.

yep, i know. you won’t get it in any ways. at least you get transparency and you can regulate sound
